### PR TITLE
docs: added docs about extensions

### PR DESCRIPTION
**Pull request by makhnatkin**

### CODE REVIEW

**Review by d3m1d0v — COMMENTED** [review]
👍 

Plz, use [the conventional names of PRs and commits for docs](https://www.conventionalcommits.org), e.g. `docs: added docs about extensions`

File: README.md [hunk 1]
```diff
@@ -46,9 +46,16 @@ function Editor({onSubmit}) {
...
+  return <MarkdownEditorView stickyToolbar autofocus toaster={toaster} editor={editor} />;
 }
 ```
+Read more:
+- [How to connect the editor in the Create React App](https://github.com/gravity-ui/markdown-editor/blob/main/docs/how-to-add-editor-with-create-react-app.md)
+- [How to add preview for markup mode](https://github.com/gravity-ui/markdown-editor/blob/main/docs/how-to-add-preview.md)
+- [How to add HTML extension](https://github.com/gravity-ui/markdown-editor/blob/main/docs/how-to-connect-html-extension.md)
+- [How to add Latex extension](https://github.com/gravity-ui/markdown-editor/blob/main/docs/how-to-connect-latex-extension.md)
+- [How to add Mermaid extension](https://github.com/gravity-ui/markdown-editor/blob/main/docs/how-to-connect-mermaid-extension.md)
+- [How to write extension](https://github.com/gravity-ui/markdown-editor/blob/main/docs/how-to-create-extension.md)
```
[the remark this answers is not in the document]
Comment (makhnatkin) [reply README.md]: fixed

File: docs/how-to-add-editor-with-create-react-app.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+npm install @craco/craco
+```
+2. Create a file called craco.config.js in the root of the project and add the following configuration:
+
+```javascript
+module.exports = {
+  webpack: {
+    configure: (webpackConfig) => {
+      webpackConfig.resolve.fallback = {
+          fs: false
```
[the remark this answers is not in the document]
Comment (makhnatkin) [reply docs/how-to-add-editor-with-create-react-app.md]: fixed

File: docs/how-to-add-editor-with-create-react-app.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+```
+
+### 4. Configuring Webpack 5 with Create React App
+To work with Webpack 5, install and configure the following packages:
+
+#### a. Installing process, path, fs:
+
+```bash
+npm install process path fs
+```
```
[the remark this answers is not in the document]
Comment (makhnatkin) [reply docs/how-to-add-editor-with-create-react-app.md]: fixed and added https://github.com/diplodoc-platform/transform/issues/473

File: docs/how-to-add-preview.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+                    'yfm',
+                    linksVisited ? 'yfm_links-visited' : undefined,
+                    noListReset ? 'yfm_no-list-reset' : undefined,
+                    className,
+                )}
+                data-qa={qa}
+            />
+        );
+    },
+);
```
[the remark this answers is not in the document]
Comment (makhnatkin) [reply docs/how-to-add-preview.md]: fixed

File: docs/how-to-connect-html-extension.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+
+```ts
+import {
+  mYfmHtmlBlockButton,
+} from '@gravity-ui/markdown-editor/bundle/config/markup';
+
+import {
+  wYfmHtmlBlockItemData,
+} from '@gravity-ui/markdown-editor';
+// ...
```
[the remark this answers is not in the document]
Comment (makhnatkin) [reply docs/how-to-connect-html-extension.md]: added more code

File: docs/how-to-connect-latex-extension.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+```ts
+import {
+  mMathListItem,
+} from '@gravity-ui/markdown-editor/bundle/config/markup';
+
+import {
+  wMathBlockItemData,
+  wMathInlineItemData,
+} from '@gravity-ui/markdown-editor';
+// ...
```
[the remark this answers is not in the document]
Comment (makhnatkin) [reply docs/how-to-connect-latex-extension.md]: added more code

File: docs/how-to-connect-mermaid-extension.md [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+
+```ts
+import {
+  mMermaidButton,
+} from '@gravity-ui/markdown-editor/bundle/config/markup';
+
+import {
+  wMermaidItemData,
+} from '@gravity-ui/markdown-editor';
+// ...
```
[the remark this answers is not in the document]
Comment (makhnatkin) [reply docs/how-to-connect-mermaid-extension.md]: added more code

File: docs/how-to-create-extension.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+        state.write('```');
+        state.ensureNewLine();
+      },
+      view: nodeView,
+    }));
+};
+
+```
+
+#### 2. Add NodeView
```
[the remark this answers is not in the document]
Comment (makhnatkin) [reply docs/how-to-create-extension.md:62]: fixed

File: docs/how-to-create-extension.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+// ...
+
+export class WMermaidNodeView implements NodeView {
+  // ...
+}
+```
+
+See the [full example of the extension](https://github.com/gravity-ui/markdown-editor/tree/main/src/extensions/yfm/Mermaid/MermaidNodeView) for more details.
+
+#### 3. Add Plugins
```
[the remark this answers is not in the document]
Comment (makhnatkin) [reply docs/how-to-create-extension.md:77]: fixed

**Review by d3m1d0v — APPROVED** [review]

**Pull request merged**